edge-modules/TwinTester/Settings.cs

```diff
@@ -48,19 +54,22 @@ public class Settings
             string serviceClientConnectionString,
             string storagePath,
             bool storageOptimizeForPerformance,
-            TestMode testMode)
+            TestMode testMode,
+            Option<string> trackingId)
         {
             this.DeviceId = Preconditions.CheckNonWhiteSpace(deviceId, nameof(deviceId));
             this.ModuleId = Preconditions.CheckNonWhiteSpace(moduleId, nameof(moduleId));
+            this.TargetModuleId = Preconditions.CheckNonWhiteSpace(moduleId, nameof(targetModuleId));
             this.TwinUpdateSize = Preconditions.CheckRange(twinUpdateSize, 1);
             this.TwinUpdateFrequency = Preconditions.CheckNotNull(twinUpdateFrequency);
             this.TwinUpdateFailureThreshold = Preconditions.CheckNotNull(twinUpdateFailureThreshold);
             this.TransportType = Preconditions.CheckNotNull(transportType);
-            this.AnalyzerUrl = Preconditions.CheckNotNull(analyzerUrl, nameof(analyzerUrl));
+            this.ReporterUrl = Preconditions.CheckNotNull(analyzerUrl, nameof(analyzerUrl));
```

Comment:
Also confused why we are using the analyzerUrl for the reporterUrl when we have a reporterUrl variable above

Reply:
I renamed to reporterUrl. The idea is that reporterUrl is the value that is passed by analyzerUrl or testResultCoordinatorUrl env variable.